File: jacoco-modules-example/core.test/src/test/java/org/example/CalculatorTest.java
package org.example;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class CalculatorTest {

    @Test
    void should_add() {
        Calculator calculator = new Calculator();
        assertEquals(4, calculator.add(2, 2), "2 + 2 should equal 4");
    }

}

File: jacoco-kotlin-example/src/test/kotlin/CalculatorTest.kt
package org.example;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class CalculatorTest {

    @Test
    fun `should add`() {
        val calculator = Calculator();
        Assertions.assertEquals(4, calculator.add(2, 2), "2 + 2 should equal 4");
    }

}
